src/extension.ts

```diff
@@ -68,11 +68,12 @@ export async function activate(context: ExtensionContext) {
   detectLaunchConfigurationChanges();
   checkServerVersion();
 
-  getJavaHome()
+  getJavaHome(outputChannel)
```

Comment:
`outputChannel` seems unused by `getJavaHome`

```suggestion
  getJavaHome()
```